src/python/aqueduct_executor/operators/connectors/tabular/spec.py

```diff
@@ -49,14 +49,20 @@ def unwrap_connector_config(cls, connector_config, values):  # type: ignore
     if not isinstance(connector_config, dict):
         raise ValueError("connector_config is not a dictionary.")
 
-    if "conf" not in connector_config:
-        # There is no inner `conf` dictionary to unwrap
-        # This occurs when the spec is serialized in Python to run operators
-        # on other engines.
+    if type(values["connector_name"]) == dict:
```

Comment:
what are these for?

Reply:
For the other jobs' connector_configs, it is directly the config whereas for the DeleteSavedObjects job, the config is actually inside a mapping with the key of the integration.